docs: use teleport systemd include for start

stevenGravy opened the pull request:
Use include to make `systemctl` instructions consistent.

stevenGravy committed:
docs: use teleport systemd include for start

stevenGravy committed:
docs: use systemctl start include

Commit:
patch broken include

Commit:
copy edits

alexfornuto reviewed (approved):
Thanks @stevenGravy! I made a couple copy edits, and fixed a different partial call I noticed while reviewing.

**Edit:** Also rebased to resolve merge conflict.

stevenGravy commented:
> Thanks @stevenGravy! I made a couple copy edits, and fixed a different partial call I noticed while reviewing.
> 
> **Edit:** Also rebased to resolve merge conflict.

Awesome, thanks!
